### DISCUSSION TITLE
私有化部署，无法获取验证码邮件

### DISCUSSION THREAD

**Discussion by micahjiang2008** [illacloud/illa-builder#413] [General]
v1.2.4 私有化部署，发送验证码邮件，返回错误信息如下
errorMessage:"send verification code error: failed to send verification code email"
UI没有任何提示，无法获得验证码邮件。

**Comment by naj1n** [comment]
@micahchiang2022 please provide a little bit more details on this error you got. What API request did ILLA Builder run? And what this request outputs did you receive? Thanks

**Comment by karminski** [comment]
@micahchiang2022 

我们已经发布了 v1.4.0 包含了大量体验提升和bugfix, 请使用 [illacli](https://github.com/illacloud/illa) 或者 [手动部署脚本](https://github.com/illacloud/deploy-illa-manually/blob/main/docker/README.md)再试一试。

非常期待您的回复。

**Reply by victoryzzg** [reply]
我在本地windows安装了illa，通过localhost:10000打开后让登录，用已经注册的账号，无法登录，F12看到调的localhost:9999的接口。
注册新账号，发送验证码失败。

**Comment by karminski** [comment]
@micahchiang2022 @victoryzzg 我们已经更新了最新的self-host版本, 该版本自带了默认用户供试用. 请使用 illa cli 部署: https://github.com/illacloud/illa
